posix.mak

```diff
@@ -40,15 +40,16 @@ DFLAGS += -w -de
 DUBFLAGS = --arch=$(subst 32,x86,$(subst 64,x86_64,$(MODEL)))
 
 TOOLS = \
-    $(ROOT)/rdmd \
-    $(ROOT)/ddemangle \
     $(ROOT)/catdoc \
+    $(ROOT)/checkwhitespace \
+    $(ROOT)/ddemangle \
     $(ROOT)/detab \
+    $(ROOT)/rdmd \
     $(ROOT)/tolf
 
 CURL_TOOLS = \
```

Comment:
AFAICT is curl bundled with Phobos since a long time, I assume it's due to the CURL linker hack that was used here before.
For reference, the hack has been removed in https://github.com/dlang/tools/pull/239, so we could remove the category `CURL_TOOLS`?